src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Resources/Bundle/BundleHandlerTests.cs

```diff
@@ -229,138 +224,6 @@ public async Task GivenABundle_WhenMultipleRequests_ReturnsABundleResponseWithCo
-                Assert.Equal("Patient?identifier=12345", references.First().Reference);
-
-                await _bundleHandler.ResolveBundleReferences(entry, referenceIdDictionary, CancellationToken.None);
-
-                // Asserting the resolved reference value after resolution
-                Assert.Equal("Patient/123", references.First().Reference);
-            }
-        }
-
-        [Fact]
-        public async Task GivenATransactionBundleWithConditionalReferences_WhenNotResolved_ThenRequestNotValidExceptionShouldBeThrown()
-        {
-            var requestBundle = Samples.GetJsonSample("Bundle-TransactionWithConditionalReferenceInResourceBody");
-            var bundle = requestBundle.ToPoco<Hl7.Fhir.Model.Bundle>();
-
-            SearchResultEntry mockSearchEntry = GetMockSearchEntry("123", KnownResourceTypes.Patient);
-            SearchResultEntry mockSearchEntry1 = GetMockSearchEntry("123", KnownResourceTypes.Patient);
-
-            var expectedMessage = "Given conditional reference 'Patient?identifier=12345' does not resolve to a resource.";
-
-            var searchResult = new SearchResult(new[] { mockSearchEntry, mockSearchEntry1 }, new Tuple<string, string>[0], Array.Empty<(string parameterName, string reason)>(), null);
-            _searchService.SearchAsync("Patient", Arg.Any<IReadOnlyList<Tuple<string, string>>>(), CancellationToken.None).Returns(searchResult);
-
-            var referenceIdDictionary = new Dictionary<string, (string resourceId, string resourceType)>();
-            foreach (var entry in bundle.Entry)
-            {
-                var exception = await Assert.ThrowsAsync<RequestNotValidException>(() => _bundleHandler.ResolveBundleReferences(entry, referenceIdDictionary, CancellationToken.None));
-                Assert.Equal(exception.Message, expectedMessage);
-            }
-        }
-
-        [Fact]
-        public async Task GivenATransactionBundleWithInvalidResourceTypeInReference_WhenExecuted_ThenRequestNotValidExceptionShouldBeThrown()
-        {
-            var requestBundle = Samples.GetJsonSample("Bundle-TransactionWithInvalidResourceType");
-            var bundle = requestBundle.ToPoco<Hl7.Fhir.Model.Bundle>();
-
-            var expectedMessage = "Resource type 'Patientt' in the reference 'Patientt?identifier=12345' is not supported.";
-
-            var referenceIdDictionary = new Dictionary<string, (string resourceId, string resourceType)>();
-            foreach (var entry in bundle.Entry)
-            {
-                var exception = await Assert.ThrowsAsync<RequestNotValidException>(() => _bundleHandler.ResolveBundleReferences(entry, referenceIdDictionary, CancellationToken.None));
-                Assert.Equal(exception.Message, expectedMessage);
-            }
-        }
-
-        [Fact]
-        public async Task GivenAConfigurationEntryLimit_WhenExceeded_ThenBundleEntryLimitExceededExceptionShouldBeThrown()
```

Comment:
Did we delete this test by mistake? It is not related to resolvingreference?

Reply:
Good catch. There were some merge conflicts from the master, resolved them and now this test is here.